spirv_cross_util.hpp

```diff
@@ -0,0 +1,28 @@
+/*
+* Copyright 2015-2018 ARM Limited
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#ifndef SPIRV_CROSS_UTIL_HPP
+#define SPIRV_CROSS_UTIL_HPP
+
+//#include "spirv_common.hpp"
+#include "spirv_cross.hpp"
+
+namespace spirv_cross_util
+{
+	void rename_interface_variable(spirv_cross::Compiler &compiler, const std::vector<spirv_cross::Resource>& resources, uint32_t location, const std::string &name);
```

Comment:
No indent here. Please use `./format_all.sh` to format this automatically.

Reply:
I'll fix the indent manually, VS keeps automatically aligning those.

Side note: `./format_all.sh` modifies quite a few files if I use the clang-format included with the latest LLVM 7 Windows binaries. It seems like `FixNamespaceCommentsL false`  is needed, and it realigns curly braces in a couple areas (I think they followed `typedef struct` and `extern "C"`). Is there a certain version I need to use?